bioblend/_tests/TestGalaxyQuotas.py

```diff
@@ -0,0 +1,59 @@
+"""
+Use ``nose`` to run these unit tests.
+"""
+import time
+
+from . import GalaxyTestBase, test_util
+
+
+@test_util.skip_unless_galaxy()
+class TestGalaxyQuotas(GalaxyTestBase.GalaxyTestBase):
+
+    def setUp(self):
+        super(TestGalaxyQuotas, self).setUp()
+        # Quota names must be unique, and they're impossible to delete
+        # without accessing the database.
+        self.quota_name = 'BioBlend-Test-Quota-%s' % time.time()
```

Comment:
In `bioblend/_tests/TestGalaxyGroups.py` we use `uuid.uuid4().hex` instead of `time.time()`, I suppose it's better for random names.